### PR TITLE
feat: EVER-13448: Add input.Checkbox to RoverUI

### PR DESCRIPTION
**Pull request by pixelbandito**

### CODE REVIEW

File: src/components/Input/Checkbox/story.tsx [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+import { Wrap } from '../../../stories/storybook-helpers';
+
+interface InteractiveInputProps
+  extends React.InputHTMLAttributes<HTMLInputElement> {
+  onChange: () => void;
+  InputRenderer: React.FC<React.InputHTMLAttributes<HTMLInputElement>>;
+}
+
+const InteractiveInput: React.FC<InteractiveInputProps> = ({
```
Comment (pixelbandito) [inline src/components/Input/Checkbox/story.tsx]: Maybe I should promote this to a test util? I can see using it a lot when writing stories for input components.
This was copy-paste with mods from the original, in the `Input/story.tsx` file.

File: src/components/Input/Checkbox/story.tsx [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+  .addParameters({
+    readme: {
+      sidebar: Readme,
+    },
+  })
+  .add(
+    'Overview',
+    () => (
+      <Wrap>
+        {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
```
Comment (pixelbandito) [inline src/components/Input/Checkbox/story.tsx:25]: I should add a comment explaining why I disabled a lint rule here.

File: src/components/Input/Checkbox/story.tsx [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+            <div className="my-6" key={['atts', ...attributes].join('-')}>
+              <label htmlFor={['atts', ...attributes].join('-')}>
+                {attributes.join(' and ') || 'default'}{' '}
+                <InteractiveInput
+                  InputRenderer={Checkbox}
+                  id={['atts', ...attributes].join('-')}
+                  onChange={action(
+                    `onChange ${attributes.join(' and ') || 'default'}`
+                  )}
+                  {...attributes.reduce(
```
Comment (pixelbandito) [inline src/components/Input/Checkbox/story.tsx]: Not super legible. Maybe I should write a helper for this? Or just not do immediate function return, so I can give this generated block of truthy props a proper name.

The upshot is: For an array like `['checked', 'disabled']`, it should produce a props value like `{ checked: true, disabled: true }`, so I can produce stories for a whole buncha variants easily.

File: src/components/Input/Input.module.css [hunk 4]
```diff
@@ -10,10 +10,10 @@
   padding: var(--rvr-space-bordered-sm);
   height: calc(var(--rvr-line-height-lg) + 2 * var(--rvr-space-sm));
   transition:
-    100ms linear background-color,
-    100ms linear border-color,
-    100ms linear box-shadow,
-    100ms linear color;
+    var(--rvr-transition-duration-fast) var(--rvr-linear) background-color,
```
Comment (pixelbandito) [inline src/components/Input/Input.module.css:13]: No-op cleanup.

File: src/components/Input/Input.module.css [hunk 5]
```diff
@@ -54,8 +54,9 @@
 
 .Input:disabled,
 .Input.disabled {
-  border-color: var(--rvr-gray-40);
-  color: var(--rvr-gray-40);
+  border-color: var(--rvr-color-disabled);
+  box-shadow: none;
```
Comment (pixelbandito) [inline src/components/Input/Input.module.css:58]: This was a UI bug! Focused "faux disabled" inputs still had a glow.

File: src/shared/colors.css [hunk 6]
```diff
@@ -98,4 +98,5 @@
   --rvr-color-chart-yellow-lite: var(--rvr-yellow-alt-lite-1);
   --rvr-color-font-dark: var(--rvr-gray-90);
   --rvr-color-font: var(--rvr-gray-80);
+  --rvr-color-disabled: var(--rvr-gray-40);
```
Comment (pixelbandito) [inline src/shared/colors.css:101]: Thought it deserved a semantic color.

File: src/shared/sizing.css [hunk 7]
```diff
@@ -64,6 +64,9 @@
   --rvr-tooltip-border-radius: 3px;
   --rvr-tooltip-offset: 20px;
 
+  /* Form elements */
+  --rvr-checkbox-size: calc(var(--rvr-baseSize * 2.25));
```
Comment (pixelbandito) [inline src/shared/sizing.css]: Add comment with evaluated size.

**Review by pixelbandito — COMMENTED** [review]
Note to self

**Review by wendigolabs — APPROVED** [review]
🌮

**Pull request merged**